2021/08/25 17:16:11 [error] 7#7: *1 open() "/etc/nginx/html/swagger/index.html" failed (2: No such file or directory), client: 172.19.0.1, server: , request: "GET /swagger/index.html HTTP/1.1", host: "localhost"
2021/08/25 17:17:01 [error] 7#7: *1 open() "/etc/nginx/html/portal-finops" failed (2: No such file or directory), client: 172.19.0.1, server: , request: "GET /portal-finops HTTP/1.1", host: "localhost"
2021/08/25 17:21:33 [warn] 7#7: *10 an upstream response is buffered to a temporary file /var/cache/nginx/proxy_temp/1/00/0000000001 while reading upstream, client: 172.19.0.1, server: , request: "GET /vendor.js HTTP/1.1", upstream: "http://172.19.0.2:4200/vendor.js", host: "localhost", referrer: "http://localhost/"
2021/08/25 18:33:56 [error] 7#7: *1 open() "/etc/nginx/html/sockjs-node/info" failed (2: No such file or directory), client: 172.19.0.1, server: , request: "GET /sockjs-node/info?t=1629916436426 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2021/08/25 18:34:01 [error] 7#7: *1 open() "/etc/nginx/html/sockjs-node/info" failed (2: No such file or directory), client: 172.19.0.1, server: , request: "GET /sockjs-node/info?t=1629916441437 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2021/08/25 18:34:10 [error] 7#7: *1 open() "/etc/nginx/html/sockjs-node/info" failed (2: No such file or directory), client: 172.19.0.1, server: , request: "GET /sockjs-node/info?t=1629916450444 HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2021/08/25 18:35:07 [error] 7#7: *1 open() "/etc/nginx/html/styles.css" failed (2: No such file or directory), client: 172.19.0.1, server: , request: "GET /styles.css HTTP/1.1", host: "localhost", referrer: "http://localhost/home"
2021/08/25 18:35:07 [error] 7#7: *2 open() "/etc/nginx/html/runtime.js" failed (2: No such file or directory), client: 172.19.0.1, server: , request: "GET /runtime.js HTTP/1.1", host: "localhost", referrer: "http://localhost/home"
2021/08/25 18:35:07 [error] 7#7: *2 open() "/etc/nginx/html/polyfills.js" failed (2: No such file or directory), client: 172.19.0.1, server: , request: "GET /polyfills.js HTTP/1.1", host: "localhost", referrer: "http://localhost/home"
2021/08/25 18:35:07 [error] 7#7: *1 open() "/etc/nginx/html/scripts.js" failed (2: No such file or directory), client: 172.19.0.1, server: , request: "GET /scripts.js HTTP/1.1", host: "localhost", referrer: "http://localhost/home"
2021/08/25 18:35:07 [error] 7#7: *6 open() "/etc/nginx/html/main.js" failed (2: No such file or directory), client: 172.19.0.1, server: , request: "GET /main.js HTTP/1.1", host: "localhost", referrer: "http://localhost/home"
2021/08/25 18:35:07 [error] 7#7: *7 open() "/etc/nginx/html/vendor.js" failed (2: No such file or directory), client: 172.19.0.1, server: , request: "GET /vendor.js HTTP/1.1", host: "localhost", referrer: "http://localhost/home"
2021/08/26 13:22:09 [warn] 7#7: *8 an upstream response is buffered to a temporary file /var/cache/nginx/proxy_temp/1/00/0000000001 while reading upstream, client: 172.19.0.1, server: , request: "GET /vendor.js HTTP/1.1", upstream: "http://172.19.0.3:4200/vendor.js", host: "localhost", referrer: "http://localhost/"
2021/08/26 19:24:06 [error] 7#7: *109 upstream prematurely closed connection while reading response header from upstream, client: 172.19.0.1, server: , request: "POST /sockjs-node/209/1tmer1p0/xhr?t=1630005843256 HTTP/1.1", upstream: "http://172.19.0.3:4200/sockjs-node/209/1tmer1p0/xhr?t=1630005843256", host: "localhost", referrer: "http://localhost/cloud-provider"
2021/08/26 19:24:19 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET /cloud-provider HTTP/1.1", upstream: "http://172.19.0.3:4200/cloud-provider", host: "localhost"
2021/08/26 19:24:19 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET /sockjs-node/info?t=1630005857171 HTTP/1.1", upstream: "http://172.19.0.3:4200/sockjs-node/info?t=1630005857171", host: "localhost", referrer: "http://localhost/cloud-provider"
2021/08/26 19:24:19 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.19.0.3:4200/favicon.ico", host: "localhost", referrer: "http://localhost/cloud-provider"
2021/08/26 19:24:21 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET /cloud-provider HTTP/1.1", upstream: "http://172.19.0.3:4200/cloud-provider", host: "localhost"
2021/08/26 19:24:21 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.19.0.3:4200/favicon.ico", host: "localhost", referrer: "http://localhost/cloud-provider"
2021/08/26 19:24:21 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET /cloud-provider HTTP/1.1", upstream: "http://172.19.0.3:4200/cloud-provider", host: "localhost"
2021/08/26 19:24:22 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.19.0.3:4200/favicon.ico", host: "localhost", referrer: "http://localhost/cloud-provider"
2021/08/26 19:24:22 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET /cloud-provider HTTP/1.1", upstream: "http://172.19.0.3:4200/cloud-provider", host: "localhost"
2021/08/26 19:24:22 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.19.0.3:4200/favicon.ico", host: "localhost", referrer: "http://localhost/cloud-provider"
2021/08/26 19:24:22 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET /cloud-provider HTTP/1.1", upstream: "http://172.19.0.3:4200/cloud-provider", host: "localhost"
2021/08/26 19:24:23 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.19.0.3:4200/favicon.ico", host: "localhost", referrer: "http://localhost/cloud-provider"
2021/08/26 19:24:23 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET /cloud-provider HTTP/1.1", upstream: "http://172.19.0.3:4200/cloud-provider", host: "localhost"
2021/08/26 19:24:23 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.19.0.3:4200/favicon.ico", host: "localhost", referrer: "http://localhost/cloud-provider"
2021/08/26 19:25:24 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET /cloud-provider HTTP/1.1", upstream: "http://172.19.0.3:4200/cloud-provider", host: "localhost"
2021/08/26 19:25:24 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.19.0.3:4200/favicon.ico", host: "localhost", referrer: "http://localhost/cloud-provider"
2021/08/26 19:25:25 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET /cloud-provider HTTP/1.1", upstream: "http://172.19.0.3:4200/cloud-provider", host: "localhost"
2021/08/26 19:25:25 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.19.0.3:4200/favicon.ico", host: "localhost", referrer: "http://localhost/cloud-provider"
2021/08/26 19:25:25 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET /cloud-provider HTTP/1.1", upstream: "http://172.19.0.3:4200/cloud-provider", host: "localhost"
2021/08/26 19:25:25 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.19.0.3:4200/favicon.ico", host: "localhost", referrer: "http://localhost/cloud-provider"
2021/08/26 19:25:26 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET /cloud-provider HTTP/1.1", upstream: "http://172.19.0.3:4200/cloud-provider", host: "localhost"
2021/08/26 19:25:26 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.19.0.3:4200/favicon.ico", host: "localhost", referrer: "http://localhost/cloud-provider"
